custom/icds_reports/utils/__init__.py

```diff
@@ -662,6 +662,10 @@ def percent(x, y):
     return "%.2f %%" % (percent_num(x, y))
 
 
+def percent_or_not_entered(x, y):
+    return percent(x, y) if y else 'data not entered'
```

Comment:
Should this use the constant for `DATA_NOT_ENTERED` instead?

Reply:
That's right, have changed that.